### PR TITLE
Firebase Analytics & Performance Interop

### PR DESCRIPTION
**Pull request by JohnMusleh**
I wanted to use firebase analytics in my flutter web project so I added it here

Note: I only implemented the functions I used
I'm new to this, so I would love to get feedback about my code (especially if I did something wrong)

Thanks :)

### CODE REVIEW

**Comment by JohnMusleh** [comment]
@googlebot I signed it!

thanks

**Comment by JohnMusleh** [comment]
I also added firebase performance interop
added to changelog @kevmoo

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## 6.0.0
-
+- Added measurementId and appId to `FirebaseOptions` (both required for analytics)
```
Comment (kevmoo) [inline CHANGELOG.md]: Please create and add this to `## 6.1.0`

Comment (JohnMusleh) [reply CHANGELOG.md]: I've done this, now there's some conflicts in pubspec and changelog
when i forked the repository v7.0.0 was not there

**Comment by jlubeck** [comment]
Can you include an example of how to use this? 
I implemented it like this:

```
import 'package:firebase/firebase.dart';

class Firebase {
  static void sendPageView(String screenName) {
    analytics().setCurrentScreen(screenName);
    print(screenName);
  }
}
```

but even though I see the debug messages from `print` I don't get anything on the firebase console.

Thanks!

**Pull request merged**